src/webgpu/capability_info.ts

```diff
@@ -490,7 +490,7 @@ export function storageTextureBindingTypeInfo(d: GPUStorageTextureBindingLayout)
   };
 }
 /** List of all GPUStorageTextureAccess values. */
-export const kStorageTextureAccessValues = ['write-only'] as const;
+export const kStorageTextureAccessValues = ['read-only', 'read-write', 'write-only'] as const;
```

Comment:
This will affect other tests (well, one test) that are parameterized on `kStorageTextureAccessValues`. Please verify that that test is still working as expected in Chrome (just to make sure the test is still correct when this happens).
<https://gpuweb.github.io/cts/standalone/?q=webgpu:api,validation,createBindGroupLayout:visibility,VERTEX_shader_stage_storage_texture_access:*>

Reply:
I've rebased my PR which makes this not needed anymore.